doc/installationguide.md

```diff
@@ -211,6 +226,10 @@ used for the same purpose. For instance:
     the IoTAgent runs in a single thread. For more details about multi-core functionality, please refer to the
     [Cluster](https://nodejs.org/api/cluster.html) module in Node.js and
     [this section](howto.md#iot-agent-in-multi-thread-mode) of the library documentation.
+-  **fallbackTenant** - For Linked Data Context Brokers which do not support multi-tenancy, this provides an alternative mechanism for suppling the `NGSILD-Tenant` header.
```

Comment:
```suggestion
-  **fallbackTenant** - For Linked Data Context Brokers which do not support multi-tenancy, this provides an alternative mechanism for supplying the `NGSILD-Tenant` header. Note that NGSILD-Tenant has not been yet included in the NGSI-LD standard (it has been proposed for the v1.3, but not decission has been done yet), take into account it could change.
```

What do you think?

Reply:
Yes, with two changes

been yet included   => yet been included 

for the v1.3, but not decission has been done yet. => for the next update of the standard, but the final decision has yet been confirmed)

Reply:
Upgraded:

```suggestion
-  **fallbackTenant** - For Linked Data Context Brokers which do not support multi-tenancy, this provides an alternative mechanism for supplying the `NGSILD-Tenant` header. Note that NGSILD-Tenant has not yet been included in the NGSI-LD standard (it has been proposed for the next update of the standard, but the final decision has yet been confirmed), take into account it could change.
```

Reply:
Fixed in 56ca556